Index: PKGBUILD
===================================================================
--- PKGBUILD	(revision 163647)
+++ PKGBUILD	(working copy)
@@ -7,9 +7,9 @@
 # Build order: avr-binutils -> avr-gcc -> avr-libc
 
 pkgname=avr-gcc
-pkgver=5.3.0
-pkgrel=2
-_snapshot=5-20160223
+pkgver=5.4.0
+pkgrel=1
+#_snapshot=5-20160223
 _islver=0.15
 pkgdesc='The GNU AVR Compiler Collection'
 arch=(i686 x86_64)
@@ -18,12 +18,12 @@
 depends=(avr-binutils gcc-libs libmpc)
 optdepends=('avr-libc: Standard C library for Atmel AVR development')
 options=(!emptydirs !strip)
-source=(#ftp://gcc.gnu.org/pub/gcc/releases/gcc-${pkgver}/gcc-${pkgver}.tar.bz2
-        ftp://gcc.gnu.org/pub/gcc/snapshots/${_snapshot}/gcc-${_snapshot}.tar.bz2
+source=(ftp://gcc.gnu.org/pub/gcc/releases/gcc-${pkgver}/gcc-${pkgver}.tar.bz2
+        #ftp://gcc.gnu.org/pub/gcc/snapshots/${_snapshot}/gcc-${_snapshot}.tar.bz2
         http://isl.gforge.inria.fr/isl-${_islver}.tar.bz2
         fix_instructions_out_of_range.patch
        )
-sha1sums=('13b7c4de9da654ef968e658c77e32b265aa4eb5b'
+sha1sums=('07524df2b4ab9070bad9c49ab668da72237b8115'
           '1e30e09a5fc2c9e1aa4bdb8c9c21fdff20a7cd12'
           'e1f8295d2c0507cf798a641877fdce7a69681739')
 
